### PR TITLE
Automatic decoding of Scylla Backtraces into Events

### PR DESCRIPTION
**Pull request by fruch**

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -1082,20 +1086,55 @@ def search_database_log(self, start_from_beginning=False, publish_events=True):
             if start_search_from_byte:
                 f.seek(start_search_from_byte)
             for index, line in enumerate(f, start=last_line_no):
+                m = backtrace_regex.search(line)
+                if m:
```
Comment (bentsi) [inline sdcm/cluster.py]: if m and backtraces

File: sdcm/cluster.py [hunk 2]
```diff
@@ -1082,20 +1086,55 @@ def search_database_log(self, start_from_beginning=False, publish_events=True):
...
+                return True
+            finally:
+                filter_backtraces.last_error = backtrace['event']
+        filter_backtraces.last_error = None
+
+        backtraces = filter(filter_backtraces, backtraces)
+
+        for backtrace in backtraces:
+            if backtrace['event'].backtrace:
+                output = self.remoter.run('addr2line -Cpife /usr/lib/debug/bin/scylla.debug %s' % " ".join(backtrace['event'].split('\n')), verbose=False)
```
Comment (bentsi) [inline sdcm/cluster.py]: - wrap with try except, log exception error
- store raw backtrace

File: sdcm/cluster.py (hunk 1 above)
Comment (fruch) [reply sdcm/cluster.py]: done

File: sdcm/cluster.py (hunk 2 above)
Comment (fruch) [reply sdcm/cluster.py]: done

**Comment by bentsi** [comment]
please add `hydra unit-tests` to run the unit tests

**Comment by fruch** [comment]
@bentsi I've added the 'hydra unit-test' command, I think this can be merged now

**Pull request merged**